commons/src/main/kotlin/com/simplemobiletools/commons/extensions/Context.kt

```diff
@@ -338,7 +338,7 @@ fun Context.getMediaContent(path: String, uri: Uri): Uri? {
 
 fun Context.queryCursor(
     uri: Uri,
-    projection: Array<String>,
+    projection: Array<String>? = null,
```

Comment:
is null used anywhere, or only for debugging? We should always specify what fields are we interested in, not query everything